source/include/ota.h

```diff
@@ -405,13 +405,19 @@ OtaErr_t OTA_Resume( void );
 /* @[declare_ota_resume] */
 
 /**
- * @brief OTA agent task function.
+ * @brief OTA agent event processing loop.
+ *
+ * This is the main event loop and the entry point to the OTA Task.
+ * This handles and executes state transitions until the Agent is
```

Comment:
This loop will continue to handle and execute events received for OTA Update until this tasks is terminated by the application.